### ISSUE TITLE
Interface and Properties names are not replacing 'default'

### ISSUE THREAD

**Issue by brandonmpetty** [krisztianb/typedoc-plugin-merge-modules#5]
Thank you very much for this plugin!
When I run:
`npx typedoc --pretty --mergeModulesRenameDefaults true ./src`

The plug does fix most of my 'default' naming problems, but it is not working with property names and interface names.

Interface:
```ts
export default interface CRUDServiceInterface {
```
Property... if that is what we'll call it:
```ts
function loggerFactory(): winston.Logger {
...
    return logger;
}

export default loggerFactory();
```

Fixing the interface name would be the most helpful, and straightforward.
As for the 'property', I would recommend either a lowercase of the file name.

- Example: logger

Or, something like: \<filename\>/@default

- Example: Logger.ts/@default

 <br/>

![image](https://user-images.githubusercontent.com/25997477/128619312-42423f36-fe7a-4618-9bc0-c4acce88fe15.png)

**Comment by krisztianb** [comment]
Thanks for reporting this. Strange that no one has done so yet.

The problem lies here: https://github.com/krisztianb/typedoc-plugin-merge-modules/blob/master/src/plugin.ts#L74

The interface issue should be easy to fix, not sure about the property. Will have to fiddle around with it.

**Comment by brandonmpetty** [comment]
Thanks for the reply with the code snippet.  After looking at it, I was able to fix the property name issue on my side by storing the value in a variable and returning that instead.  This removes it from the "Properties" section of the documentation and into "Variables"... with the expected name.  It also lets me control the name to some extent.

Thank you for looking into the interface issue.

**Comment by krisztianb** [comment]
Interface bug fixed in 3.0.1 🎉
